x-pack/test/api_integration/apis/entity_manager/helpers/clear_entity_definitions.ts

```diff
@@ -0,0 +1,17 @@
+/*
```

Comment:
Did you think about moving these helpers to a test service (e.g. like [this one](https://github.com/elastic/kibana/blob/main/x-pack/test/api_integration/services/fleet_and_agents.ts))? Then then you could use the `es` and `supertest` services directly and would not have to pass the client instances around.

Reply:
I did not and that's a great suggestion, I'll likely address that in a follow up